xml/tuning_storagescheduler.xml

```diff
@@ -79,14 +79,28 @@
     set by the device driver.
    </para>
   </note>
+
+  <note os="sles" arch="x86_64">
+   <title>Scheduler in case of block multi-queue (blk-mq) I/O path</title>
+   <para>
+     In case the device is using blk-mq I/O path (refer to <xref
+     linkend="cha-tuning-io-scsimq"/>)
+     <replaceable>SCHEDULER</replaceable> is one of
+     <option>mq-deadline</option>, <option>kyber</option>,
+     <option>bfq</option>, or <option>none</option>
+   </para>
+  </note>
+
  </sect1>
  <sect1 xml:id="cha-tuning-io-schedulers">
   <title>Available I/O Elevators</title>
 
   <para>
-   In the following elevators available on &productname; are listed. Each
-   elevator has a set of tunable parameters, which can be set with the
-   following command:
+   In the following elevators available on &productname; are listed
```

Comment:
"In the following" is meant as "in what follows" or "below". What was meant before my change is that in the following subsections elevators are listed which are available on this SUSE product. That didn't change (and so was confusing already?). I've just added a distinction between elevators for "legacy block layer" and those for blk-mq.

Yes, I mean adjustable. Was the original sentence less confusing?: "Each elevator has a set of tunable parameters, which can be set with the following command:"
I think I've modified this because "none" and "noop" options for elevator don't have tunable parameters. State of affairs is: If an elevator (i/O scheduler) has tunable parameters, they can be change with the given command.

Reply:
I see. How about this then?

_Below is a list of elevators available on &productname;. Each elevator has a set of tunable parameters that can be set with the for devices that use the legacy block I/O path._

Reply:
I'd say it should be changed to:

"Below is a list of elevators available on &productname; for devices that use the legacy block I/O path. If an elevator has tunable parameters, they can be set with the command:"

I am not sure how to add such a suggestion to batch of changes (using github) and maybe its best to put all suggestions that required additional discussion and were not commited immediately in a follow-up patch on top of the changes committed after the pull request review. What do you think?

Reply:
I like your suggestion. Let's go with it. I send you an email with instructions how to proceed.